chain/ethereum.go

```diff
@@ -6,9 +6,8 @@ import (
 	"math/big"
 	"strings"
 
-	"github.com/nakji-network/connector/common"
-
 	"github.com/ethereum/go-ethereum"
+	ethcommon "github.com/ethereum/go-ethereum/common"
```

Comment:
Could you remove the alias?
```suggestion
	"github.com/ethereum/go-ethereum/common"
```